lib/pinboard_tabs_downloader.rb

```diff
@@ -0,0 +1,89 @@
+require 'fileutils'
+require 'pathname'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.